src/schema/v2/schema.ts

```diff
@@ -245,6 +245,15 @@ const OptionalFieldDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD],
 })
 
+const CacheableDirective = new GraphQLDirective({
+  name: "cacheable",
+  locations: [
+    DirectiveLocation.FIELD,
+    DirectiveLocation.FRAGMENT_SPREAD,
+    DirectiveLocation.QUERY,
+  ],
+})
```

Comment:
I know this is spike-y but can we still follow existing decorator conventions? src/directives

Reply:
all our directives are vanilla and defined right here in this file (scroll a few lines above this)

Reply:
(also thinking this should _only_ apply at the query level, that ensures someone has 'checked' the whole query and tagged it as such)

Reply:
Ah, then that's a bit of tech debt. We should move that definition into https://github.com/artsy/metaphysics/blob/main/src/directives/optionalField/optionalFieldsDirectiveExtension.ts, etc